examples/message/src/configureStore.js

```diff
@@ -0,0 +1,20 @@
+import {applyMiddleware, createStore} from 'redux'
+import {createLogger} from 'redux-logger'
+
+const reducer = (state, action) => {
```

Comment:
This should be in `reducers.js`

We need to create a structure at least similar to conventional Redux structure, so that it's more comprehensive for redux users.